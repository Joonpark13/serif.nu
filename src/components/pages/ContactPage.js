import React from 'react';
import PropTypes from 'prop-types';
import { withStyles } from '@material-ui/styles';
import { pageContainer } from './common/styles';
import PageTitle from './common/PageTitle';

export const styles = {
  pageContainer,
};

function ContactPage({ classes }) {
  return (
    <div className={classes.pageContainer}>
      <PageTitle title="Contact Us" />
    </div>
  );
}

ContactPage.propTypes = {
  classes: PropTypes.objectOf(PropTypes.string).isRequired,
};

export { ContactPage as UnstyledContactPage };
export default withStyles(styles)(ContactPage);
