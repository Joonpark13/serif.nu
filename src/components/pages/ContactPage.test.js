import React from 'react';
import { shallow } from 'enzyme';
import ContactPage from './ContactPage';

describe('ContactPage', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<ContactPage />);

    expect(wrapper.get(0)).toMatchSnapshot();
  });
});
