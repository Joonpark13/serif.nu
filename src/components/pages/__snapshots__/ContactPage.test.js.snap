// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactPage should render correctly 1`] = `
<div
  className=""
>
  <ForwardRef
    title="Contact Us"
  />
</div>
`;
